app/__init__.py

```diff
@@ -33,6 +34,18 @@
 from .blueprints import gitlab, graph, jupyterhub, notebooks, webhooks
 from .config import config
 
+# Wait for the VS Code debugger to attach if requested
+VSCODE_DEBUG = os.environ.get('VSCODE_DEBUG') == "1"
+print("{}:{}".format(os.environ.get('VSCODE_DEBUG'), VSCODE_DEBUG))
```

Comment:
This will evaluate to `None:False` if the env variable is not set.

Reply:
Sorry, that was some left-over logging. Thanks for fixing the problem with the tests.